src/oci.c

```diff
@@ -561,6 +562,11 @@ cc_oci_create (struct cc_oci_config *config)
 		return false;
 	}
 
+	if (! cc_oci_ns_setup (config)) {
```

Comment:
`cc_oci_ns_setup()` is currently called in `cc_oci_vm_launch()` (which is called soon after `cc_oci_handle_mounts()` below). Can we therefore drop the call in `cc_oci_vm_launch()` ?